Cargo.toml

```diff
@@ -23,6 +23,8 @@ members = [
     "common/num-variants",
     "common/proptest-helpers",
     "common/proxy",
+    "common/rate-limiter",
+    "common/rate-limiter/integration-test",
```

Comment:
You shouldn't need to add this as a top level crate, see 
https://github.com/diem/diem/blob/master/consensus/safety-rules/Cargo.toml#L49
that points to https://github.com/diem/diem/blob/master/consensus/safety-rules/tests/binary.rs